charts/internal/calico-monitoring/calico-typha-dashboard.json

```diff
@@ -0,0 +1,2360 @@
+{
```

Comment:
None of the panels in this dashboard have descriptions. Can you add descriptions to the panels?